### DISCUSSION TITLE
List of classes

### DISCUSSION THREAD

**Discussion by aaronjensen** [tailwindlabs/tailwindcss#2872] [Help]
Hi all,

I'm curious if there is a way to get a list of available classes for tailwind from the combination of a tailwind config and the `tailwindcss` package. I've briefly looked at the way that tailwindcss-intellisense does it and it's a little hard to understand, plus it's async and intended to be used in a language server. I'm looking to do it one off so that I can have a list with which to statically verify classes that are used.

Thanks!

**Comment by ghost** [comment]
Parsing raw file? (prob should be a better way)
https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css

**Reply by aaronjensen** [reply]
Yeah, but that doesn’t allow for any configuration. That’s just the default classes afaict

**Comment by Airkro** [comment]
same request:

https://github.com/avencera/rustywind/issues/14
